vehicle/raw_vehicle_cmd_converter/CMakeLists.txt

```diff
@@ -1,57 +1,35 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
+
 project(raw_vehicle_cmd_converter)
-add_compile_options(-std=c++14)
-
-find_package(
-  catkin REQUIRED COMPONENTS
-  roscpp
-  std_msgs
-  autoware_vehicle_msgs
-  geometry_msgs
-)
 
-catkin_package(
-  INCLUDE_DIRS include
-  LIBRARIES accel_map_converter
-  CATKIN_DEPENDS roscpp std_msgs autoware_vehicle_msgs geometry_msgs
-)
+### Compile options
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
 
 include_directories(
```

Comment:
I didn't need this `include_directories` anymore when using `ament_*`. Try it out

Reply:
Thanks! I didn't realize that.